lib/fizzbuzz.rb

```diff
@@ -1,9 +1,15 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    if (input % 3 == 0) && (input % 5 == 0) 
+      "FizzBuzz"
+  	elsif input % 5 == 0
+  	  "Buzz"
+    elsif input % 3 == 0
+  	  "Fizz"
+  	else
+  	  input
+  	end 
   end
```

Comment:
Thanks for pointing out indentation errors - it's been mentioned before and
I though it was me not being careful, but I've updated my Sublime Text
settings and I hope my github repos will reflect what I am actually seeing
locally.

nate

On Fri, Jan 15, 2016 at 3:05 PM, Matt DeGoey <notifications@github.com>
wrote:

> In lib/fizzbuzz.rb
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/40#discussion_r49898810>
> :
>
> > @@ -1,9 +1,15 @@
> >  class SuperFizzBuzz
> >
> >    def run(input)
> > -
> > -    #Implement your code here
> > -
> > +    if (input % 3 == 0) && (input % 5 == 0)
> > +      "FizzBuzz"
> > +  	elsif input % 5 == 0
> > +  	  "Buzz"
> > +    elsif input % 3 == 0
> > +  	  "Fizz"
> > +  	else
> > +  	  input
> > +  	end
> >    end
>
> Please make your indentation for each if, elsif, and else consistent.
> Initially this made me think the first elsif would only hit if the first if
> evaluated to true for both checks.
>
> Also, you don't need parenthesis around separate compare statements. Still
> works with them there, but I believe the convention is to not use them,
> especially for such small and simple comparisons.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/40/files#r49898810>
> .
>